### ISSUE TITLE
Implement page for Regional Coordinator bios

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/learning.mozilla.org#2280]
Implementation of the project #1815.
Here is the final mockups: https://redpen.io/p/nj4f7768ca3e7b79cf

**Comment by taisdesouzalessa** [comment]
Assets coming soon

**Comment by taisdesouzalessa** [comment]
@gvn Here is the mockup in Sketch: 
https://drive.google.com/a/mozilla.com/file/d/0B3b1KWRwjxtoT1JmZ0tjZkE0Q1E/view?usp=sharing

Here are the assets (pictures):
https://drive.google.com/drive/folders/0B3b1KWRwjxtocUU2dmN4YkJhRGs?usp=sharing

The map marker icon is a Glyphicon from Bootstrap (glyphicon glyphicon-map-marker) and the Twitter icon is in Font Awesome (fa-twitter [&#xf099;]) - please let me know if you need svgs for those icons.

**Comment by taisdesouzalessa** [comment]
2 interaction notes:
1. Could you please make the tab "Start a Club" inactive once the user clicks on the link ("featured Regional Coordinators") and goes to the next page? So they have a visual clue that the tab is clickable again if they want to return to the page they were. Here is a gif showing that:
![tabbed-behaviour](https://cloud.githubusercontent.com/assets/22157921/19005297/a205d262-870e-11e6-88ba-67713b1715b1.gif)

2. Is it possible to give an id or name in the html to each person so we have a direct url for each name? Like:
http://example.com#name1
http://example.com#name2
The goal is for the Regional Coordinators be able to give an URL that shows their name in first place without the need to scroll.

Let me know if you have any questions. Thanks!

**Comment by gvn** [comment]
I don't think we should de-activate the tab once you navigate to the regional coordinators view. If it's a sub-view of "Start a Club", then we should probably use a breadcrumb to indicate that. By de-activating the tab you're disconnecting the view from the visible navigation hierarchy. The view becomes an "orphan".

If regional coordinators is not supposed to be a sub-view of the "Start a Club" tab, then it should probably be its own tab instead.

FWIW: This is the same point that I'm raising in #2320.

**Comment by gvn** [comment]
Also, regarding your 2nd question, yes we should be able to set anchor IDs for each person.

**Comment by taisdesouzalessa** [comment]
Thanks for your insights @gvn. It makes sense to me. @sabrinang @flukeout could we meet next week to discuss alternatives? The issue is the same here and in the Gigabit site, as Gavin pointed out so one solution will fit both.

**Comment by flukeout** [comment]
@taisdesouzalessa Did you mock up some options with breadcrumbs before? If you have, can you post them? Happy to discuss anytime.

**Comment by taisdesouzalessa** [comment]
No, unfortunately  have no mockups with that treatment. I think @sabrinang had some options with breadcrumbs for Gigabit. Sabrina, remember we talked about this and you showed me some options? Do you still have those mockups with breadcrumbs? If not, I can quickly do some for our discussion later on.
